spec/adama/errors_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper'
+
+RSpec.describe Adama::Errors::BaseError do
+  describe '#to_s' do
+    subject(:error_string) { described_class.new(error: error, command: command).to_s }
+    before(:all) { Object.const_set('TestCommand', Class.new.send(:include, Adama::Command)) }
```

Comment:
ruby complains if you mutate a constant so `before(:all)` ensures we set the constant just once.